src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/AuthenticationTests.cs

```diff
@@ -43,19 +47,21 @@ public async Task DefaultAddressTest()
             
             await toolRunner.StartAsync(DefaultTimeout);
 
-            using ApiClient client = new(_outputHelper, await toolRunner.GetDefaultAddressAsync(DefaultTimeout));
+            using HttpClient httpClient = _httpClientFactory.CreateClient();
```

Comment:
I'm thinking I can create an extension method that combines creation to the http client and setting the base address in order to minimize the boilerplate code.